packages/kbn-i18n/src/index.ts

```diff
@@ -22,3 +22,4 @@ import * as loader from './loader';
 
 export const i18n = i18nCore;
 export const i18nLoader = loader;
+export { PlainMessages } from './messages';
```

Comment:
__note:__ We'll eventually rename `PlainMessages` to something that is clearer in one of the upcoming PRs (I believe there is one is ready for review already).